samples/asgardeo-react-ts-app/src/app.tsx

```diff
@@ -40,15 +40,29 @@ const SecureRouteWithRedirect: FunctionComponent<{component: any, path: string,
     return (<SecureRoute exact path={ path } component={ component } callback={ callback } />);
 };
 
+const AppContent: FunctionComponent = (): ReactElement => {
+    const { error } = useAuthContext();
+
+    const systemDateTimeError: string = error?.code === "JS-CRYPTO_UTILS-IVIT-IV02";
+    
+    {
+        return systemDateTimeError
+            ? <div>ERROR with system date</div>
```

Comment:
Shall we add a friendly message like this?
![tttt](https://user-images.githubusercontent.com/25959096/165225795-1c9e67c7-2edc-4886-b1a4-137558f49fa4.png)